client/src/app/game/board/board-row.tsx

```diff
@@ -27,34 +32,54 @@ const getSlotByLocation = (
 
 type BoardRowProps = {
 	type: 'left' | 'right'
-	player?: PlayerEntity
+	boardForPlayerEntity: PlayerEntity
```

Comment:
This name is a bit confusing

Reply:
I am going to disagree with you because "boardForPlayerEntity" is very specific, but if you elaborate on what you think is confusing I can adjust it.